connector-sdk/core/src/main/java/io/camunda/connector/api/inbound/CorrelationResult.java

```diff
@@ -40,28 +42,54 @@ record MessageAlreadyCorrelated(@JsonIgnore ProcessElementContext activatedEleme
 
   sealed interface Failure extends CorrelationResult {
 
+    default CorrelationFailureHandlingStrategy handlingStrategy() {
+      return ForwardErrorToUpstream.RETRYABLE;
+    }
+
+    /**
+     * @deprecated Use {@link #handlingStrategy()} instead
+     */
+    @Deprecated(forRemoval = true)
     default boolean isRetryable() {
       return false;
     }
```

Comment:
Perhaps we can remove it right away without deprecation? I'm not sure if there is someone out there implementing and using it 🙂

Reply:
If they do, it means they have built their own runtime, isn't it? If so, let's just remove it 🔫

Reply:
Not necessarily, they could have just used it in custom connectors, but I'd say the change is not very invasive.